app/components/Sidebar/index.tsx

```diff
@@ -195,6 +196,23 @@ const Sidebar = (): JSX.Element => {
                 </Typography>
               </Link>
             </ListItem>
+            <ListItem
```

Comment:
I can see this is just copied from the existing pattern, so no need to do in this PR. but all of these list items have a lot of shared logic so I think it would be beneficial to extract that into a small component

Reply:
https://www.bcgsc.ca/jira/browse/DEVSU-1788